### PR TITLE
OZ-162: Add Option to run behind Traefik

### PR DESCRIPTION
**Pull request by enyachoke**
The ticket for this work: https://mekomsolutions.atlassian.net/browse/OZ-162

### CODE REVIEW

**Review by imbuchi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ Type in the following in a terminal:
 ```bash
 git clone https://github.com/ozone-his/ozone-docker
 cd ozone-docker
+# Create the web Docker network if it does not exist yet.
+# The project is configured to run behind Traefik running in the web Docker network but the demo uses Apache.
+# So you have to create this network for Docker Compose to work.
+# For running behind Traefik see the manual instructions.
```
Comment (mks-d) [inline README.md:21]: This is too verbose for a quick start, I would suggest not writing any of this and just make sure that we come up with a compact set of instructions that work for everyone.

Such details need to be provided in the manual setup guide.

File: readme/manual-setup.md [hunk 2]
```diff
@@ -89,9 +90,48 @@ If you are doing development on Ozone and are building the Ozone distro in your
 ```bash
 export DISTRO_PATH=/your/path/to/ozone-distro/target/ozone-distro-$VERSION
 ```
+### Step 6. Setting up Traefik
```
Comment (mks-d) [inline readme/manual-setup.md:93]: So I made this its own step in the setup, however, unlike with the quick start, it is never required to run this:
```bash
docker network create web
```
Is that intended?

File: README.md [hunk 3]
```diff
@@ -43,6 +43,17 @@ Ozone FOSS requires you to log into each component separately:
...
+For macOS replace 172-17-0-1 with your host IP. For example if your host IP is 192.168.200.197 replace with 192-168-200-197
+
+| HIS Component     | URL                            | Username | Password |
+|-------------------|--------------------------------|----------|----------|
+| OpenMRS 3         | htts://emr-172-17-0-1.traefik.me  | admin    | Admin123 |
+| OpenMRS Legacy UI | htts://emr-172-17-0-1.traefik.me/openmrs   | admin    | Admin123 |
+| SENAITE           | htts://lims-172-17-0-1.traefik.me | admin    | password |
+| Odoo              | htts://erp-172-17-0-1.traefik.me       | admin    | admin    |
+| Superset          | htts://analytics-172-17-0-1.traefik.me       | admin    | password |
+
```
Comment (mks-d) [inline README.md:56]: Nope, the quick start guide needs to lead to **one** clear outcome, options and choice subtleties are left to the manual setup.
(Btw you have left a typo in each of the Traefik URLs.)

So, we need to choose what the quick start achieves by default, and make it do just that.

**Review by mks-d — CHANGES_REQUESTED** [review]

**Comment by rbuisson** [comment]
@enyachoke , is that a breaking change for people who currently deploy with Ozone Docker?

**Comment by enyachoke** [comment]
> [enyachoke](https://github.com/enyachoke) , is that a breaking change for people who currently deploy with Ozone Docker?
Yes, we just can't have support for both so they will need to make some changes.

**Comment by rbuisson** [comment]
In that case, we need to change the version number isn't it? I'm a bit concerned that all of a sudden the Ozone Distro Kh deployments will stop working.
I'm not saying we should release `1.0.0` though. Might want to keep a `1.x` branch for reference and upgrade to `2.0.0-SNAPSHOT` directly.

Does that sounds OK @enyachoke ?

**Pull request merged**